sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_with_batch_transform.ipynb

```diff
@@ -325,16 +331,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "prediction = probabilities.index(max(probabilities))\n",
-    "print('Prediction is {}'.format(prediction))"
+    "print(predictions)"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "conda_mxnet_p36",
+   "display_name": "conda_tensorflow_p36",
```

Comment:
Why use the TensorFlow kernel here instead of MXNet?  I guess neither of the deep learning libraries are really needed by the notebook itself, but since it's an MXNet notebook, opening in the TensorFlow kernel could be confusing for customers.

Reply:
you're right, that was my bad. (I wrote these with a different kernel entirely and wasn't thinking when I added these back in.) fixed.